cocli/cmd/corimExtract.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim/corim"
+)
+
+var (
+	corimExtractCorimFile *string
+	corimExtractOutputDir *string
+)
+
+var corimExtractCmd = NewCorimExtractCmd()
+
+func NewCorimExtractCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "extract",
+		Short: "extract CoSWIDs and CoMIDs from a CoRIM and save them to disk",
```

Comment:
In both lines 27 and 28 we should clarify extract CoSWIDs and CoMIDs as CBOR from a CoRIM and save them to disk

Reply:
Also wondering from a user perspective, it will really add value, if we can extract the CoMID and CoSWID in a user supplied directory as JSON ? Also then it will help the use understand quickly what was packed inside the  CoMID/CoSWID? Just a suggestion, to think about!